### ISSUE TITLE
Issues with Creating a New Sequence

### ISSUE THREAD

**Issue by RichardRennII** [smeighan/xLights#50]
When creating a new sequence with the new sequence wizard.

 If you try to add 25,50,100 ms timings the label will show but the actual marks will not.

When trying to import timing marks I get an error can't open file the error reads "(error 0; the operation completed successfully.) and the label or timing marks do not import. 

When saving the new sequence I am getting the same error as importing timing marks.

After closing out 4.0 and re-opening it I select open sequence, the sequence did save as a fseq, but when it opens there are no models and or timing tracks. 

Grid does not scroll with play

**Comment by AzGilrock** [comment]
I'm pretty sure the error 0 issue was corrected when I fixed Issue #47 but you probably don't have that fix in your current build.

For the timing marks  showing are you saying they don't show up on the row beside the label or they don't show up vertically across the grid because we only display them vertically for one timing group at a time.  You click the radio button by the name to display them.  If it's not showing up on the row then I'm not sure because my checkin on Feb 25th had that working.  I don't know if the code pulled for your build included that checkin.

Grid not scrolling is a completely different issues.  It's best not to group several differing issues together because as developers fix things we like to close the issue with our checkin and if they are grouped we can't do that if we aren't fixing all the problems at once.

**Comment by RichardRennII** [comment]
Sorry for that Gil I will not group the issues from this point on.

timing marks are not showing up across the grid I am getting the label the
states 50 ms but there are no timings in the grid but that is only on a new
sequence.

**Comment by smeighan** [comment]
I also went througgh a process of creating a new sequence. i said i wanted 50ms timing.
I see a label called 50ms, but no timing marks in the grid.
I manually added some timing marks to a track called "new".

Here is snippet from xml file

<ElementEffects>
    <Element type="timing" name="50ms">
      <EffectLayer/>
    </Element>
    <Element type="timing" name="new">
      <EffectLayer>
        <Effect label="" startTime="0.000000" endTime="78.025000"></Effect>
        <Effect label="" startTime="78.025000" endTime="78.525000"></Effect>
        <Effect label="" startTime="78.525000" endTime="78.950000"></Effect>
        <Effect label="" startTime="78.950000" endTime="79.375000"></Effect>
        <Effect label="" startTime="79.375000" endTime="79.725000"></Effect>
        <Effect label="" startTime="79.725000" endTime="80.450000"></Effect>
        <Effect label="" startTime="80.450000" endTime="81.125000"></Effect>
        <Effect label="" startTime="81.125000" endTime="81.675000"></Effect>
        <Effect label="" startTime="81.675000" endTime="82.000000"></Effect>
      </EffectLayer>
    </Element>

**Comment by AzGilrock** [comment]
Yeah I know why this is happening but need to think about how to fix it.  There was actually a crash that was happening when you tried to add timings to a new sequence because the sequencer grid has not been loaded yet.  There is a circular issue for me to figure out.  The XML logic requires the sequencer to exist to load the timings but the sequencer grid needs the xml object to initialize.

**Comment by AzGilrock** [comment]
For me this is working fine for the very first New Sequence I create.  I did run into the "error 0" if I try to open a second New Sequence and click save so I'll look into that.

Sean I don't know how you got the XML you are showing me.  For one the logic to save the XML file for a new sequence still had the bug where it doesn't actually create a file so I just got that fixed.  Also any fixed timing element should have the fixed="50" attribute.  Here's what I got when I created and saved a new sequence.  Maybe if you could do a Pull and try to reproduce what happened last time but there definitely seems to be an issue when a second sequence is created and saved.
```xml
<DisplayElements>
     <Element name="50ms" active="0" visible="1" type="timing" collapsed="0"/>
</DisplayElements>
<ElementEffects>
     <Element name="50ms" type="timing" fixed="50">
          <EffectLayer/>
     </Element>
</ElementEffects>
```

**Comment by AzGilrock** [comment]
I checked in a fix for the "error 0" I was getting.

I'm not closing this yet.  Need someone to confirm whether original problems exist after all the latest checkins are included.

**Comment by RichardRennII** [comment]
After downloading the latest release I have not gotten the "error 0 " and it imports the timings I have created fine but will still not display 25,50 or 100 ms timings to a new sequence

**Comment by AzGilrock** [comment]
Ok I've tried everything you stated causes the crash above.  The only thing I found was a crash if you attempted to add a fixed timing track that was a duplicate like you already have 50ms and try to add a second one.  I've fixed that and checked it in just now so get the next build and try to cause this crash again.  If you get one I need to know exact steps to reproduce.

Also when you say the timings do not display I would like to see a screenshot.  1) Make sure the timing track is selected or else it won't show the timings.  2) If you don't have Aero turned off the timings may not show and that is documented in issue #58.
